packages/components/addon/components/hds/reveal/index.hbs

```diff
@@ -0,0 +1,21 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: MPL-2.0
+}}
+
+<Hds::DisclosurePrimitive class={{this.classNames}} @isOpen={{@isOpen}} ...attributes>
+  <:toggle as |t|>
+    <Hds::Reveal::Toggle::Button
+      aria-controls={{this.contentId}}
+      @text={{if (and t.isOpen @textWhenOpen) @textWhenOpen @text}}
+      @isOpen={{t.isOpen}}
+      {{on "click" t.onClickToggle}}
+    />
+  </:toggle>
+
+  <:content>
+    <div class="hds-reveal__content hds-typography-body-300" id={{this.contentId}}>
```

Comment:
@KristinLBradley I see you're using the term "content" here, but in the `Tabs` component, we used the term "panel." Should these be consistent?

Reply:
To me, the content which is hidden and shown by `Reveal` is more generic so just using "content" makes sense. (In `Tabs` there are a series of "panels" which are hidden and shown vs. one chunk of "content" being toggled between hiding/showing) Also it fits the naming used by the primitive as @didoo said.

Reply:
Should `Tabs` be updated to "content" then? :homerfade:

Reply:
No, `Tabs` is a more complex component and there are a series of full width "panels" that are controlled by activating the individual tabs. `Reveal` is much simpler and just toggles on and off the display of generic content. The content isn't full width and doesn't swap display between other content.

At least that's how I think of it...